financial-connections/src/main/java/com/stripe/android/financialconnections/HostedAuthUrlBuilder.kt

```diff
@@ -55,6 +56,9 @@ internal object HostedAuthUrlBuilder {
             phoneCountryCode?.let { queryParams.add("linkMobilePhoneCountry=$it") }
         }
 
+        // hint for frontend to understand the surface launching the web flow.
+        queryParams.add("launched_by=android_sdk")
```

Comment:
As part of this PR we're adding this hint for frontend to know the source launching the hosted auth url CC @mats-stripe